index.html

```diff
@@ -16,7 +16,7 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="./index.css" />
-    <title>Document</title>
+    <title>Music Association of Uganda</title>
```

Comment:
Great design @Zeph180 🤝

- [ ] [Optional] Kindly use the correct font as provided in the project requirements. Fonts used originally: Cocogoose, Lato. This will make your text a replica of the given design.
- Your design.
![yours](https://user-images.githubusercontent.com/63917725/212040593-14e85377-9203-4d57-99a9-c861152b5ccf.PNG)

- Given design
![Given](https://user-images.githubusercontent.com/63917725/212040649-331ec2ae-88a1-4b24-977d-7891f496c102.PNG)